nautobot/docs/macros.py

```diff
@@ -0,0 +1,13 @@
+"""mkdocs-macros-plugin data loading etc."""
+
+import os.path
+
+import yaml
+
+
+def define_env(env):
+    settings_file = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "nautobot", "core", "settings.yaml"
+    )
+    with open(settings_file, "rt") as fh:
+        env.variables["settings_data"] = yaml.safe_load(fh)
```

Comment:
Consider changing this to `"settings_schema"` here and in the markdown files for clarity?